indico_zodbimport/modules/event_papers.py

```diff
@@ -169,6 +223,87 @@ def _migrate_paper_roles(self, contribution):
             elif role.role == PaperReviewingRoleType.editor:
                 contribution.paper_layout_reviewers.add(role.user)
 
+    def _migrate_review(self, contribution, old_judgment, review_type):
+        # Consider legacy custom states the same as "to be corrected"
+        proposed_action = JUDGMENT_STATE_PAPER_ACTION_MAP.get(int(old_judgment._judgement._id),
+                                                              PaperAction.to_be_corrected)
+        review = PaperReview(user=old_judgment._author.user, comment=old_judgment._comments, type=review_type,
+                             proposed_action=proposed_action, created_dt=_to_utc(old_judgment._submissionDate))
+        for old_answer in old_judgment._answers:
+            old_question = old_answer._question
+            question = self.legacy_question_map.get(old_question, None)
+            if not question:
+                self.importer.print_warning(cformat('%{yellow!}Answer to deleted question {} has been ignored [{}, {}]')
+                                            .format(old_question._id, contribution, review_type),
+                                            event_id=self.event.id)
+                continue
+
+            rating = PaperReviewRating(question=question, value=old_answer._rbValue)
```

Comment:
Added an assert, it seems to be fine. Also noticed I was just saving the stored value (0-6) instead of using `-3 ... 3` as it is displayed to the user.